user-guide/Cloud_Platform/CloudAdminApp/Granting_admin_consent.md

```diff
@@ -42,8 +42,12 @@ To grant admin consent and configure your tenant:
 
    After you log in successfully, your tenant is configured.
 
+
+> [!NOTE]
+> You can revoke the permissions given to Skyline at any time in the Azure Portal. For more information, refer to the [Microsoft documentation](https://docs.microsoft.com/en-us/azure/active-directory/manage-apps/manage-application-permissions?pivots=portal).
+
+> [!NOTE]
+> You can unlink the tenant by clicking the x next to the tenant ID. However, unlinking the tenant from the organization **will not remove the permissions given to Skyline**. See the note above on how to revoke the given permissions.
+
 > [!NOTE]
```

Comment:
If there are several notes in a row, we always combine this in one note with bullet points. I'll adjust this in the review.

Reply:
To me it was more clear to see them as separate notes, otherwise i would have the intention to just easily skip them if they are too big/long to read. But alright :)